api/.env.example

```
@@ -37,3 +37,6 @@ LOG_LEVEL=debug
 RESEND_API_KEY=
 RESEND_FROM_EMAIL=noreply@justadrop.xyz
 RESEND_FROM_NAME=Just a Drop
+
+
+X_AUTH_ID=this_is_x_auth_id
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)